### PR TITLE
[plugin.audio.radiothek] 0.1.2

### PR DESCRIPTION
**Pull request by s0faking**
### Description
fixed broken livestreams, added livestream settings, added sort on podcast views
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.audio.radiothek/resources/lib/Addon.py [hunk 1]
```diff
@@ -166,7 +167,7 @@ def get_input():
 def searchHistoryPush(title):
     if not title:
         return
-    addonUserDataFolder = xbmc.translatePath("special://profile/addon_data/plugin.audio.radiothek")
+    addonUserDataFolder = xbmcvfs.translatePath("special://profile/addon_data/plugin.audio.radiothek")
```
Comment (enen92) [inline plugin.audio.radiothek/resources/lib/Addon.py]: This is not compatible with leia

File: plugin.audio.radiothek/resources/lib/Addon.py [hunk 2]
```diff
@@ -191,7 +192,7 @@ def searchHistoryPush(title):
 
 
 def searchHistoryGet():
-    addonUserDataFolder = xbmc.translatePath("special://profile/addon_data/plugin.audio.radiothek")
+    addonUserDataFolder = xbmcvfs.translatePath("special://profile/addon_data/plugin.audio.radiothek")
```
Comment (enen92) [inline plugin.audio.radiothek/resources/lib/Addon.py]: Same

**Review by enen92 — CHANGES_REQUESTED** [review]

File: plugin.audio.radiothek/resources/lib/Helpers.py [hunk 3]
```diff
@@ -23,6 +24,13 @@ def parameters_string_to_dict(parameters):
     return param_dict
 
 
+def radiothek_log(msg, debug=False):
```
Comment (enen92) [inline plugin.audio.radiothek/resources/lib/Helpers.py]: sorry debug must be the default

**Review by enen92 — APPROVED** [review]

**Comment by enen92** [comment]
@s0faking can you rebase?

**Comment by s0faking** [comment]
> @s0faking can you rebase?

@enen92 do you mean "Resolve conflicts"? when i did that the last time i had 2 commits :( sorry my git skills are quite limited ;)

**Comment by enen92** [comment]
No problem with having two commits, we can then squash them

**Comment by enen92** [comment]
```
ERROR: plugin.audio.radiothek addon already exists with a lower or equal version: 0.1.2 in matrix branch and the addon has non forward abi compatible dependencies. Users migrating to kodi version matrix won't be able to receive the addon update.
```

**Comment by s0faking** [comment]
> ```
> ERROR: plugin.audio.radiothek addon already exists with a lower or equal version: 0.1.2 in matrix branch and the addon has non forward abi compatible dependencies. Users migrating to kodi version matrix won't be able to receive the addon update.
> ```

sorry for the holdup. i already pushed a matrix version. i thought that i have to push one for matrix & leia because otherwise users with 18.x wouldnt get that update. or did i just push the updates in the wrong order?

**Comment by enen92** [comment]
@s0faking sorry for the delay. It looks the wrong order, the version in matrix should be always higher than the one in leia. So for this one to be valid, the version in matrix should be > 0.1.2, not equal. What you can do to keep the "same" versions is to append a localversion identifier to the matrix version. So:

matrix: 0.1.2+matrix.1
leia: 0.1.2

that way matrix > leia although the "same" version

**Comment by s0faking** [comment]
> @s0faking sorry for the delay. It looks the wrong order, the version in matrix should be always higher than the one in leia. So for this one to be valid, the version in matrix should be > 0.1.2, not equal. What you can do to keep the "same" versions is to append a localversion identifier to the matrix version. So:
> 
> matrix: 0.1.2+matrix.1
> leia: 0.1.2
> 
> that way matrix > leia although the "same" version

i see. so i need to push a new matrix version before proceeding with the leia version right? on https://kodi.wiki/view/Addon.xml it says that the version attribute in the addon.xml should only contain text if its a beta version. is it ok anyway to add "~matrix" to the tag?

**Comment by enen92** [comment]
Yes, use x.x.x+matrix.1

**Pull request merged**